pkg/api/python/api.proto

```diff
@@ -0,0 +1,407 @@
+syntax = "proto3";
```

Comment:
You should use the existing proto file. https://github.com/kubeflow/katib/blob/master/pkg/api/api.proto

Reply:
Yes, the current `api_pb2.py` and `api_pb2_grpc.py` is generated using the original proto file mentioned above. Let me delete the unused one:)